docs/hadoop_streaming_hue.md

```diff
@@ -0,0 +1,131 @@
+Big data within [Hadoop](http://hadoop.apache.org/) clusters is mainly analyzed by means of the [MapReduce](https://en.wikipedia.org/wiki/MapReduce) paradigm. Usually, specific Java jobs are created for this purpose, but sometimes different programming languages may want to be used, such as Python or even Shell scripting. In that cases, [Hadoop Streaming](https://hadoop.apache.org/docs/current/hadoop-streaming/HadoopStreaming.html) is the feature the IoT Platform integrators need.
```

Comment:
Maybe we should start to introduce a bit of strcuture on .md files. It is expected that with the per-topic approach its number grows, making just one flat directory difficult to manage.

Maybe we can start with a docs/topic/ directory to hold this (and future) topics .md files.